packages/sheets/src/commands/mutations/set-worksheet-col-width.mutation.ts

```diff
@@ -26,6 +26,18 @@ export interface ISetWorksheetColWidthMutationParams {
     colWidth: number | IObjectArrayPrimitiveType<Nullable<number>>;
 }
 
+export interface ISetWorksheetColIsAutoWidthMutationParams {
+    unitId: string;
+    subUnitId: string;
+    ranges: IRange[];
+}
+
+export interface ISetWorksheetColAutoWidthMutationParams {
```

Comment:
These interfaces seem not used by other code so it is better to remove them.

Reply:
It used by  export interface ISetWorksheetColAutoWidthMutationParams in this file